doc/integrator/multi_organization.rst.tmpl

```diff
@@ -0,0 +1,102 @@
+. _integrator_multi_organization:
+
+Multi organization
+==================
+
+Our application can host multiple organizations with small changes between organization.
```

Comment:
Suggestion:
"The geoportal can host multiple organizations, with configuration differences for each organization. In a multi-organization geoportal, each organization will have the same program code (Python, Javascript, SCSS)."